core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraSession.kt

```diff
@@ -532,7 +533,7 @@ private suspend fun startVideoRecordingInternal(
         when (onVideoRecordEvent) {
             is VideoRecordEvent.Finalize -> {
                 when (onVideoRecordEvent.error) {
-                    ERROR_NONE ->
+                    ERROR_NONE, ERROR_DURATION_LIMIT_REACHED ->
```

Comment:
I think it needs to handle an `ERROR_DURATION_LIMIT_REACHED` condition separate from an `ERROR_NONE`.
Such as show a message to user or else.

Reply:
I didn't think it would need any special treatment because this is expected behavior. 

`ERROR_DURATION_LIMIT_REACHED` is not like the other errors. It is a **successful** recording. It is different from `ERROR_NONE` in that that the video was stopped because it reached its max duration, and not because the user manually stopped it.

Reply:
I see. Thank you.